pkgs/servers/oauth2_proxy/default.nix

```diff
@@ -2,16 +2,18 @@
 
 buildGoModule rec {
   pname = "oauth2-proxy";
-  version = "5.1.1";
+  version = "6.0.0";
+
+  goPackagePath = "github.com/oauth2-proxy/${pname}";
```

Comment:
```suggestion
```

`buildGoModule` does not take a `goPackagePath`.

Reply:
Thanks, fixed!